git_externals/git_externals.py

```diff
@@ -350,14 +357,16 @@ def autosvn_update_checkout(reset):
         mkdir_p(externals_root_path())
         with chdir(externals_root_path()):
             repo_name = get_repo_name(normalized_ext_repo)
+            ext_name = git_externals[ext_repo].get('name', '')
```

Comment:
Si, ma solo nel caso in qui `externals = load_gitexts()` ritorna qualcosa perché senno `get_repo_name` ritorna un `name` *come prima* (dedotto da `repo` passato a `get_repo_name`). Ti sto solo leggendo il codice perché non mi ricordo esattamente perché era cosi. L'unica cosa che mi ricordo è che diventa complicato quando ci sono nested externals, sopra tutto perché non c'e una struttura dati che contiene l'albero intero degli externals, ma invece, i file externals.json sono letti e riletti e devi stare attento a dove ti trovi (working dir).